lib/libzpool/zrepl_mgmt.c

```diff
@@ -545,10 +545,10 @@ uzfs_zvol_destroy_snapshot_clone(zvol_state_t *zv, zvol_state_t *snap_zv,
 	    clone_zv->zv_name, clonename, zv->zv_name);
 
 	/* Destroy clone's snapshot */
-	ret = uzfs_destroy_internal_all_snap(clone_zv);
+	ret = uzfs_destroy_all_internal_snapshots(clone_zv);
 	if (ret != 0) {
 		LOG_ERR("Rebuild_clone snap destroy failed on:%s"
-		    " with err:%d", clone_zv->zv_name, ret);
+		    " with err:%d", zv->zv_name, ret);
 	}
 
 	uzfs_zvol_release_internal_clone(zv, snap_zv, clone_zv);
```

Comment:
if clone_zv is NULL, there can be panic in this release_internal_clone.

Reply:
ok